dev/tests/integration/testsuite/Magento/CatalogImportExport/Model/Import/ProductTest.php

```diff
@@ -741,7 +741,7 @@ protected function getOptionValues(\Magento\Catalog\Model\Product\Option $option
     /**
      * Test that product import with images works properly
      *
-     * @magentoDataIsolation enabled
+     * @magentoDbIsolation enabled
```

Comment:
So, it didn't have effect before? Why do we need to enable DB isolation then?

Reply:
@orlangur updated.